static/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -356,6 +384,9 @@ class Actions extends Component<Props, State> {
     const isResolved = status === 'resolved';
     const isIgnored = status === 'ignored';
 
+    // TODO: In the future we will be able to access a 'type' property on groups, we should use that instead
```

Comment:
So i think we'll actually have the `type` property very soon.

What I'm thinking is, instead of doing this in a one off kind of way, let's have a constant object that's exported somewhere that is like `issueTypeCapabilities`, where it's a mapping of the issue type, to various boolean `capabilities`, like, does it support suspect commits, deleting, merging, discarding, etc etc

Reply:
I'll create a PR for this

Reply:
That sounds good to me 👍